martini-build-image/cloudwatch.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_cloudwatch_log_group" "martini_project_log_group" {
+  name              = "/aws/codebuild/${var.environment}-${var.pipeline_name}-codebuild"
+  retention_in_days = var.log_retention_days
+  tags              = var.tags
+}
+
+resource "aws_cloudwatch_log_stream" "martini_project_log_stream" {
```

Comment:
Is it needed @jasonquinto ? If so, why? If not, why?

Reply:
I initially added this while trying to simplify the resource object instead of building the ARN. I thought permission for log groups and log streams needed to be resource-specific because I was encountering the "no identity-based policy allows the logs:CreateLogStream action" error. But it turned out that I was actually being too specific, which caused the permissions to fail to map correctly to the CloudWatch resource for the CodeBuild log group. After retesting, I replaced it with `Resource = "${aws_cloudwatch_log_group.martini_project_log_group.arn}:*"`, and it’s now working as expected.